GitUI/BranchTreePanel/RepoObjectsTree.cs

```diff
@@ -229,6 +234,27 @@ public void Initialize([CanBeNull]IAheadBehindDataProvider aheadBehindDataProvid
             _ = UICommandsSource;
         }
 
+        public void SelectionChanged(IReadOnlyList<GitRevision> selectedRevisions)
+        {
+            var cancellationToken = _selectionCancellationTokenSequence.Next();
+
+            string selectedGuid = selectedRevisions.FirstOrDefault()?.Guid;
+            ThreadHelper.JoinableTaskFactory.RunAsync(async () =>
+            {
+                cancellationToken.ThrowIfCancellationRequested();
+                var mergedBranches
+                    = selectedGuid == null ? new HashSet<string>()
```

Comment:
```suggestion
                var mergedBranches = selectedGuid == null 
                       ? new HashSet<string>()
```

Reply:
Can this be `Array.Empty<string>`?

Reply:
> Can this be `Array.Empty<string>`?

I need something for lookups. Would become `Array.Empty<string>().ToHashSet()` because `HashSet` lacks `Empty`.